src/geometries.jl

```diff
@@ -91,21 +91,39 @@ include("primitives/cylinder.jl")
 """
     Polytope{N,Dim,T}
 
-We say that a geometry is a N-polytope when it is a collection of "flat" sides that constitue a N-dimensional subspace.
-They are called polygon and polyhedron respectively for 2D (N=2) and 3D (N=3) subspaces, embedded in a `Dim`-dimensional space.
-The term polytope expresses a particular combinatorial structure. A polyhedron, for example, can be decomposed into faces. Each face can then be decomposed into edges, and edges into vertices.
-Some conventions act as a mapping between vertices and higher dimensional features (edges, faces, cells...), removing the need to store all features. We follow the [ordering conventions](https://gmsh.info/doc/texinfo/gmsh.html#Node-ordering) of the GMSH project.
+We say that a geometry is a N-polytope when it is a collection of "flat" sides
+that constitue a `N`-dimensional subspace. They are called polygon and polyhedron
+respectively for 2D (`N=2`) and 3D (`N=3`) subspaces, embedded in a `Dim`-dimensional
+space. The parameter `N` is also known as the rank or parametric dimension of the
+polytope: https://en.wikipedia.org/wiki/Abstract_polytope.
+
+The term polytope expresses a particular combinatorial structure. A polyhedron,
+for example, can be decomposed into faces. Each face can then be decomposed into
+edges, and edges into vertices. Some conventions act as a mapping between vertices
+and higher dimensional features (edges, faces, cells...), removing the need to
+store all features. We follow the ordering conventions of the GMSH project:
+https://gmsh.info/doc/texinfo/gmsh.html#Node-ordering
```

Comment:
Do you use a setup to cut lines automatically or do you do it by hand?

Reply:
I am currently doing it by hand, but VS Code definitely has an option for it. I should probably activate it to break the line after certain column.